omnibus/config/projects/chef.rb

```diff
@@ -33,13 +33,16 @@
 end
 
 override :bundler,        version: "1.10.6"
-override :ruby,           version: "2.1.6"
 
-override :'ruby-windows', version: "2.0.0-p645"
-# Leave dev-kit pinned to 4.5 because 4.7 is 20MB larger and we don't want
-# to unnecessarily make the client any fatter.
-if windows_arch_i386?
-  override :'ruby-windows-devkit', version: "4.5.2-20111229-1559"
+if windows?
+  override :'ruby-windows', version: "2.0.0-p645"
```

Comment:
The fact that if people pull in the new omnibus-software, they have to do something like this is a little scary. @schisamo do you know if anybody besides chef uses omnibus for their Windows projects?